opam-monorepo.opam.locked

```diff
@@ -462,4 +516,4 @@ x-opam-monorepo-duniverse-dirs: [
   ]
 ]
 x-opam-monorepo-root-packages: ["opam-monorepo"]
-x-opam-monorepo-version: "0.1"
+x-opam-monorepo-version: "0.2"
```

Comment:
```suggestion
x-opam-monorepo-version: "0.1"
```

This might be a strange idea, but if we just patch the lock file version here, the CI can be green without adding support to ocaml-ci right now:
- it'll install depexts (using the "wasteful" technique)
- opam monorepo pull 0.1.0 and master understand this lock file version

Reply:
Yeah let's do that and we'll update once this is released. We'll need to think of a better way to support new features before the release in the future though!